src/tools/ping/PingConnection.cpp

```diff
@@ -482,6 +492,11 @@ PingConnection::ProcessEvent(
         break;
     }
 
+    case QUIC_CONNECTION_EVENT_RESUMED: {
+        printf("[%p] Received 0-RTT resumption ticket!\n", QuicConnection);
```

Comment:
Don't print this.

Reply:
But how will I know when a session is resumed?

Reply:
Not every tool needs to inform the user. This tool allows for hundreds, if not thousands of connections to be open simultaneously. I don't want it spitting out stuff to the console for every one.

Reply:
After following up offline, I'll remove it.